templates/security/cve/cve.html

```diff
@@ -289,9 +285,9 @@ <h2>References</h2>
         <ul>
           {% if cve.references %}
             {% for reference in cve.references %}
-              {% if cve._clean_url(reference) %}
-                <li><a href="{{ cve._clean_url(reference) }}">{{ reference }}</a></li>
-              {% endif %}
+              
+                <li><a href="{{ cve.refrence }}">{{ reference }}</a></li>
```

Comment:
misspelled - does this work?

Reply:
Oh wait, shouldn't this be:

```suggestion
                <li><a href="{{ reference }}">{{ reference }}</a></li>
```

Reply:
Yup, fixed!